### PR TITLE
2021 Annual Report: SIG Cluster Lifecycle

### PR DESCRIPTION
**Pull request by neolit123**
xref https://github.com/kubernetes/community/issues/6369

### CODE REVIEW

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 1]
```diff
@@ -127,11 +152,10 @@ Operational tasks in [sig-governance.md]:
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
 - [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
```
Comment (neolit123) [inline sig-cluster-lifecycle/annual-report-2021.md:154]: these 3 need https://github.com/kubernetes/community/pull/6402 to merge.

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 2]
```diff
@@ -22,82 +32,100 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
...
 
-   -
-   -
-   -
+   Subproject that need help from more contributors:
+   - [etcdadm](https://github.com/kubernetes-etcdadm)
+   - [cluster-addons](https://github.com/kubernetes-sigs/cluster-addons)
+   - [cluster-api-provider-gcp](https://github.com/kubernetes-sigs/cluster-api-provider-gcp)
+   - [kubeadm](https://github.com/kubernetes/kubeadm)
+   - ?
```
Comment (neolit123) [inline sig-cluster-lifecycle/annual-report-2021.md:61]: should remove. no other entries were requested here.

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 3]
```diff
@@ -22,82 +32,100 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
...
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We started collecting metrics from all of our subprojects using a
+     [yearly Google Forms survey](https://forms.gle/xZn8DXww4XxPsXvCA).
```
Comment (neolit123) [inline sig-cluster-lifecycle/annual-report-2021.md:66]:
```suggestion
   - We started collecting metrics from all of our subprojects using an
     [yearly Google Forms survey](https://forms.gle/xZn8DXww4XxPsXvCA).
```

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 4]
```diff
@@ -4,15 +4,25 @@
...
-   -
-   -
-   -
+   - We started preparation for the Dockershim removal in 1.24. Minikube and kubeadm
+   maintainers helped with documentation updates, implementing changes and talking
+   to the cri-dockerd maintainers.
+   - Cluster API continues on its road to maturity and released its first MAJOR release
+   (v1.0) of the GitHub repository. The API itself graduated to v1beta1.
+   - kubeadm continues with some of the efforts to solve technical debt issues
+   and further stabilize the project. A new API version v1beta3 was released.
```
Comment (neolit123) [inline sig-cluster-lifecycle/annual-report-2021.md:13]: extract some from "Current initiatives" from the survey.

File: sig-cluster-lifecycle/annual-report-2021.md (hunk 1 above)
Comment (neolit123) [reply sig-cluster-lifecycle/annual-report-2021.md:154]: rebased / updated https://github.com/kubernetes/community/pull/6402 again.

**Comment by neolit123** [comment]
did some post survey finalization edits and cleanups.

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 5]
```diff
@@ -106,17 +152,14 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
...
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+Retired in 2021:
+- [WG Component Standard](https://github.com/kubernetes/community/commit/a8fb89db5534f659e62e5c04528445b933d8e434)
 
 Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+- NONE
```
Comment (neolit123) [inline sig-cluster-lifecycle/annual-report-2021.md:162]: forgot that we are sponsoring wg reliability:
https://github.com/kubernetes/community/blob/master/wg-reliability/README.md#stakeholder-sigs

**Comment by dims** [comment]
@neolit123 looks like there is one last thing left to do before we can start a clock and merge this? thanks!

**Comment by neolit123** [comment]
- this PR needs approval; has relevant updates:
https://github.com/kubernetes/community/pull/6458#discussion_r806004900
- add WG reliability:
https://github.com/kubernetes/community/pull/6458#discussion_r831348581

**Comment by neolit123** [comment]
> * this PR needs approval; has relevant updates:
>   [2021 Annual Report: SIG Cluster Lifecycle #6458 (comment)](https://github.com/kubernetes/community/pull/6458#discussion_r806004900)
> * add WG reliability:
>   [2021 Annual Report: SIG Cluster Lifecycle #6458 (comment)](https://github.com/kubernetes/community/pull/6458#discussion_r831348581)

@dims , updated to resolve these items ^

**Comment by dims** [comment]
/approve
/lgtm

thanks @neolit123

**Pull request merged**